students/kondel_tomasz/lesson_03_funtions/the_collatz_sequence.py

```diff
@@ -0,0 +1,27 @@
+def collatz(input_string):
+    try:
+        number = int(input_string)
```

Comment:
So it's my stupid formulation of the exercise that made you take strings here, which is stupid. Sorry for that.